utils/node_local_test.py

```diff
@@ -2563,6 +2563,13 @@ def test_uns_create(self):
         print(stbuf)
         assert stbuf.st_ino < 100
         print(os.listdir(path))
+        rc = run_daos_cmd(self.conf, ['filesystem', 'memquery', '--path', self.dfuse.dir])
```

Comment:
Should we support positional?
```
filesystem memquery <path>
```

Reply:
Probably, I'll speak to @tanabarr about how to add that.